python/mxnet/module/base_module.py

```diff
@@ -333,7 +334,7 @@ def predict(self, eval_data, num_batch=None, merge_batches=True, reset=True,
 
         Parameters
         ----------
-        eval_data : DataIter
+        eval_data : DataIter or NDArray or ndarray
```

Comment:
DataIter or NDArray or numpy array?

Reply:
yes, the current API makes it difficult for a developer to get a model up and running. given a `predict` API, I think it is reasonable for most developers to assume that it would accept a ndarray of some sort. Also our documentation currently for the API is not too instructive: https://mxnet.apache.org/api/python/module/module.html (the example on the top of the page says that it accepts a nd_iter). if you go to the documetation for predict it links to the DataIter class (https://mxnet.apache.org/api/python/io/io.html#mxnet.io.DataIter) without showing how to use it.

Reply:
got that thanks!. I think it is be useful.
I was suggesting to be explicit `evalData: DataIter or NDArray or numpy array`. I should have used a few more words to describe what i was asking :)